ecommerce/regis/static/sass/base/_navbar.scss

```diff
@@ -0,0 +1,54 @@
+// Navbar styles ecommerce/ecommerce/static/sass/partials/components/_navbar.scss.
+
+$navbar-gray: #242E33;
```

Comment:
Would you consider to remove this variable?, since "$main-elements-color" in _variables.scss has the same value

Reply:
Good catch.